ssh.sh

```diff
@@ -4,7 +4,7 @@ echo "Generating a new SSH key for GitHub..."
 
 # Generating a new SSH key
 # https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key
-ssh-keygen -t ed25519 -C $1
+ssh-keygen -t ed25519 -C $1 -f ~/.ssh/id_ed25519
```

Comment:
We could even add the passphrase if you want

```suggestion
ssh-keygen -t ed25519 -C $1 -f ~/.ssh/id_ed25519 -P $2
```

Reply:
Hmm could do that but does the script know that up front? Or will it fail at this line?

Reply:
If you pass it as an argument (just like the e-mail), it will work

Reply:
Okay feel free to pr that. Also update the docs if you can, thanks!